bin/autobahn-router/src/edge.rs

```diff
@@ -17,6 +17,7 @@ pub struct EdgeState {
     // TODO: it may be much better to store this centrally, so it's cheap to take a snapshot
     pub cached_prices: Vec<(u64, f64, f64)>,
     is_valid: bool,
+    is_dirty: bool,
```

Comment:
can you add comments clarifying dirty vs valid?

Reply:
Added in de74c464238b4. Works for you ?